applications/FemToDemApplication/custom_elements/femdem3d_element.cpp

```diff
@@ -38,12 +38,10 @@ FemDem3DElement::FemDem3DElement(IndexType NewId, GeometryType::Pointer pGeometr
 
 	// Each component == Each edge
 	mNumberOfEdges = 6;
-	mF_sigmas = ZeroVector(mNumberOfEdges);   // Equivalent stress
+	mNonConvergedThresholds = ZeroVector(mNumberOfEdges);   // Equivalent stress

```

Comment:
Most of the code in 2d and 3d is duplicated, can you simplify that?

Reply:
I know that but some parts of the algorithm are slightly differetn in 2d and 3d so it's not possible. In another PR ÇI can create a library of yield surfaces like in structural in order to simplify the elements...